content/200-concepts/100-components/01-prisma-schema/09-index-configuration.mdx

```diff
@@ -1,26 +1,38 @@
 ---
-title: 'Index configuration'
-metaDescription: 'How to configure index functionality to allow sorting and length restriction'
+title: 'Indexes'
```

Comment:
Technically this name change should probably also trigger a filename change - but that also requires to setup redirects that the old `/index-configuration` still works and redirects to the new location.

Reply:
Thanks, I'd be happy to do that, just checking how it works - is it a case of updating the redirects [here](https://github.com/prisma/website-v8/blob/main/public/_redirects) and having that go out at the same time as the PR? Is there a particular section of the redirects list it should go in?

Reply:
There is also an internal redirects system of the docs, I am not super sure how that works.

I am also not super sure where to optimally put the redirect in the website-v8, but this is a good candidate I think: https://github.com/prisma/website-v8/blob/00d90532be9d372e65f802301ed651f433c9f915/public/_redirects#L423 It is definitely next to other docs redirects that seem to be about renaming and moving pages.

Reply:
Thanks, I've changed the filename now and [put in a PR](https://github.com/prisma/website-v8/pull/402) on the website-v8 repo